test/config-luatest/credentials_applier_test.lua

```diff
@@ -754,97 +596,6 @@ g.test_remove_user_role = function(g)
     })
 end
 
-g.test_restore_defaults_for_default_user = function(g)
-    -- Verify that if the default users and roles are not present in config
-    -- their excessive privileges are revoked (restored to built-in defaults).
```

Comment:
AFAIU, this behavior should remain unchanged and it worth to test. Can we adopt the test case instead of removing it?

Reply:
This part was actually broken. Fixed.

Reply:
Nice! Thank you!